Crypto/PublicKey+Windows.cc

```diff
@@ -685,6 +693,47 @@ namespace litecore::crypto {
         return new Cert(slice(winCert->pbCertEncoded, winCert->cbCertEncoded));
     }
 
+    bool Cert::exists(PublicKey *subjectKey) {
```

Comment:
Where is this used?

Reply:
Nowhere where it’s used currently, but as there is a Cert::load function with the same parameter (as there is also a Cert::load function that takes a name parameter), I figured it may be useful to implement for future use. Of course if you think it isn’t necessary I can remove it.

Reply:
Cannot you refactor the code to avoid duplicated code with load() then?

Reply:
Yes possibly, that’s a good idea. I’ll take a look at it tomorrow.